source/_templates/cloud/python_installation.rst

```diff
@@ -0,0 +1,18 @@
+.. Copyright (C) 2024 Wazuh, Inc.
+
```

Comment:
```suggestion

The |service_module| module requires `Python 3 <https://www.python.org/>`__. Specifically, it's compatible with `Python |PYTHON_CLOUD_CONTAINERS_MIN|–|PYTHON_CLOUD_CONTAINERS_MAX| <https://www.python.org/downloads/>`_. While later Python versions should work as well, we can't assure they are compatible.

```

Reply:
In a template, `|PYTHON_CLOUD_CONTAINERS_MIN|–|PYTHON_CLOUD_CONTAINERS_MAX|` is interpreted literally and not replaced.

Reply:
Agreed, I prefer having it inside the template. I will change that.